lib/ui/window/platform_configuration.h

```diff
@@ -0,0 +1,126 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_UI_WINDOW_PLATFORM_CONFIGURATION_H_
+#define FLUTTER_LIB_UI_WINDOW_PLATFORM_CONFIGURATION_H_
+
+#include <memory>
+#include <set>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "flutter/fml/time/time_point.h"
+#include "flutter/lib/ui/semantics/semantics_update.h"
+#include "flutter/lib/ui/window/platform_message.h"
+#include "flutter/lib/ui/window/pointer_data_packet.h"
+#include "flutter/lib/ui/window/viewport_metrics.h"
+#include "flutter/lib/ui/window/window.h"
+#include "third_party/tonic/dart_persistent_value.h"
+
+namespace tonic {
+class DartLibraryNatives;
+
+// So tonic::ToDart<std::vector<int64_t>> returns List<int> instead of
+// List<dynamic>.
+template <>
+struct DartListFactory<int64_t> {
+  static Dart_Handle NewList(intptr_t length) {
+    return Dart_NewListOf(Dart_CoreType_Int, length);
+  }
+};
+
+}  // namespace tonic
+
+namespace flutter {
+class FontCollection;
+class PlatformMessage;
+class Scene;
+
+// Must match the AccessibilityFeatureFlag enum in framework.
+enum class AccessibilityFeatureFlag : int32_t {
+  kAccessibleNavigation = 1 << 0,
+  kInvertColors = 1 << 1,
+  kDisableAnimations = 1 << 2,
+  kBoldText = 1 << 3,
+  kReduceMotion = 1 << 4,
+  kHighContrast = 1 << 5,
+};
+
+class PlatformConfigurationClient {
+ public:
+  virtual std::string DefaultRouteName() = 0;
```

Comment:
These should really all have comments. It's a pre-existing issue though, and guess is should be cleaned up in a docs sweep at some point.

Reply:
Done.